components/UsMap.js

```diff
@@ -0,0 +1,130 @@
+import React, { Component } from 'react'
+import statesLived from "../public/data/stateslived.json"
+import stateScores from "../public/data/state-scores.json"
+import usData from "../public/data/us-states.json"
+import * as d3 from "d3"
+
+
+class UsMap extends Component {
+
+  componentDidMount() {
+
+
+    this.statesLived = statesLived;
+    this.stateScores = stateScores;
+    this.usData = usData;
+    this.height = 500; (64, 0, 64)
+    this.width = 960;
+    this.colorRange = [
+      // lightest
+      "rgb(168,0,168)",
+      "rgb(128,0,128)",
+      "rgb(98,0,98)",
+      "rgb(78,0,78)",
+      "rgb(58,0,58)"
+    ]
+    this.renderMap();
+  }
+
+  mapScoresToStates(scoreData, usData) {
+
+    for (var i = 0; i < scoreData.length; i++) {
+      let scoreStateName = scoreData[i].state;
+      var stateScores = scoreData[i].score;
+      for (var j = 0; j < usData.features.length; j++) {
+        var jsonStateName = usData.features[j].properties.name;
+        if (scoreStateName == jsonStateName) {
+          usData.features[j].properties.score = stateScores;
+          break;
+        }
+      }
+    }
+    return usData
+  }
+
+  renderMap() {
+
+    // just unpacking for tidier variable names downstream
+    let usData = this.usData;
+    let statesLived = this.statesLived;
+    let colorRange = this.colorRange;
+    let width = this.width;
+    let height = this.height;
+    let stateScores = this.stateScores;
+
+    // add the values to the state objects
+    // usData = this.mapStatesToValues(statesLived, usData);
+    usData = this.mapScoresToStates(stateScores, usData)
+
+    // initalize the pojection and path
+    let projection = d3
+      .geo
+      .albersUsa()
+      .translate([width / 2, height / 2])
+      .scale([1000]);
+
+    let path = d3
+      .geo
+      .path()
+      .projection(projection);
+
+    let color = d3
+      .scale
+      .linear()
+      .range(colorRange);
+
+    let tooltip = d3
+      .select('body')
+      .append('div')
+      .attr('id', 'tooltip')
+      .attr('style', 'position: absolute; opacity: 0;');
+
+    d3.select("body")
+      .append("svg")
+      .attr("width", width)
+      .attr("height", height)
+      .selectAll("path")
+      .data(usData.features)
+      .enter()
+      .append("path")
+      .attr("d", path)
+      .style("stroke", "#fff")
+      .style("stroke-width", "1")
+      .style("fill", function (d) {
+        return color(d.properties.score)
+      }).on('mouseover', function (d) {
+        d3.select(this)
+          .style({ opacity: '0.75' })
+          .append("text")
+          .text(function (d) {
+            console.log(d.properties.name)
```

Comment:
I think we can remove any `console.log` in the project we don't really need, makes a cleaner prod

Reply:
Roger!